src/map/clan.cpp

```diff
@@ -116,6 +116,12 @@ void clan_member_joined( struct map_session_data* sd ){
 
 	nullpo_retv(sd);
 
+	if( sd->clan != nullptr ){
```

Comment:
is it ok to have this before the clan lookup ?
```
clan = clan_search(sd->status.clan_id);
nullpo_retv(clan);
```
it seem we are sending data to client which might be invalid, this look a bit incoherent for me